src/epics/login/githubLogin.epic.ts

```diff
@@ -64,7 +64,7 @@ export const githubLogin = (action$: AnyAction, state$: any) => action$.pipe(
                 )),
                 switchMap((data: any) => authService.githubAuth(data)),
                 switchMap(() => userService.getUser()),
-                mergeMap((data) => [authActions.loginSuccess(data), userActions.getProjectList()]),
+                mergeMap((data) => [authActions.loginSuccess(data), authActions.refreshAuthStart(), userActions.getProjectList()]),
```

Comment:
I would not perform this action here but actually when the application starts instead.